observability/logging/logging-6.2/log6x-configuring-lokistack-otlp-6.2.adoc

```diff
@@ -0,0 +1,145 @@
+
+[id="custom-attribute-mapping-for-openshift_{context}"]
+=== Custom attribute mapping for OpenShift
+
+When using the {loki-op} in `openshift-logging` mode, attribute mapping follow OpenShift default values, but you can configure custom mappings to adjust default values. 
+In the `openshift-logging` mode, you can configure custom attribute mappings globally for all tenants or for individual tenants as needed. When you define custom mappings, they are appended to the OpenShift default values. If you do not need default labels, you can disable them in the tenant configuration.
+
+[NOTE]
+====
+A major difference between the {loki-op} and Loki lies in inheritance handling. Loki copies only `default_resource_attributes_as_index_labels` to tenants by default, while the {loki-op} applies the entire global configuration to each tenant in the `openshift-logging` mode.
+====
+
+Within `LokiStack`, attribute mapping configuration is managed through the `limits` setting. See the following example `LokiStack` configuration:
+
+[source,yaml]
+----
+# ...
+spec:
+  limits:
+    global:
+      otlp: {} # <1>
+    tenants:
+      application: <2>
+        otlp: {} # 
+----
+<1> Defines global OTLP attribute configuration.
+<2> Defines the OTLP attribute configuration for the `application` tenant within the `openshift-logging` mode. You can also configure `infrastructure` and `audit` tenants in addition to `application` tenants.
+
+[NOTE]
+====
+You can use both global and per-tenant OTLP configurations for mapping attributes to stream labels.
+====
+
+Stream labels derive only from resource-level attributes, which the `LokiStack` resource structure reflects. See the following `LokiStack` example configuration:
+
+[source,yaml]
+----
+spec:
+  limits:
+    global:
+      otlp:
+        streamLabels:
+          resourceAttributes:
+          - name: "k8s.namespace.name"
+          - name: "k8s.pod.name"
+          - name: "k8s.container.name"
+----
+
+You can drop attributes of type resource, scope, or log from the log entry.
+
+[source,yaml]
+----
+# ...
+spec:
+  limits:
+    global:
+      otlp:
+        streamLabels:
+          # ...
+        drop:
+          resourceAttributes:
+          - name: "process.command_line"
+          - name: "k8s\\.pod\\.labels\\..+"
+            regex: true
+          scopeAttributes:
+          - name: "service.name"
+          logAttributes:
+          - name: "http.route"
+----
+
+[TIP]
+====
+Use regular expressions by setting `regex: true` to apply a configuration for attributes with similar names.
+====
+
+[IMPORTANT]
+====
+Avoid using regular expressions for stream labels, as this can increase data volume.
+====
+
+Attributes that are not explicitly set as stream labels or dropped from the entry are saved as structured metadata by default.
+
+[id="customizing-openshift-defaults_{context}"]
```

Comment:
Can this be a CONCEPT module? Maybe with the title OpenShift defaults customization? Since it isn't a procedure, it doesn't really need to be a gerund form title.